File: PMUG02_16_2021_Project/Start_Java/app/src/main/java/com/neudesic/pmugfebappstart/SecondActivity.java
package com.neudesic.pmugfebappstart;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.RecyclerView;

import com.google.android.material.floatingactionbutton.FloatingActionButton;
import com.neudesic.pmugfebappstart.viewmodel.PersonViewModel;

public class SecondActivity extends AppCompatActivity {
    private RecyclerView recyclerView;
    private FloatingActionButton floatingActionButton;

    private PersonViewModel viewModel;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_second);

        recyclerView = findViewById(R.id.recycler);
        floatingActionButton = findViewById(R.id.fab);
    }
}

File: PMUG02_16_2021_Project/Start_Kotlin/app/src/main/java/com/neudesic/pmug02_16_2021_kotlin/SecondActivity.kt
package com.neudesic.pmug02_16_2021_kotlin

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.recyclerview.widget.RecyclerView
import com.google.android.material.floatingactionbutton.FloatingActionButton
import com.neudesic.pmug02_16_2021_kotlin.viewmodel.PersonViewModel

class SecondActivity : AppCompatActivity() {
    private lateinit var recycler: RecyclerView
    private lateinit var floatingActionButton: FloatingActionButton

    private lateinit var viewModel: PersonViewModel

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        recycler = findViewById(R.id.recycler)
        floatingActionButton = findViewById(R.id.fab)


    }
}